tests/foreman/ui/test_errata.py

```diff
@@ -131,11 +255,31 @@ def test_positive_apply_for_all_hosts(self):
 
         @Assert: Check that the erratum is applied in all the content hosts.
 
-        @caseautomation: notautomated
-
-
-        @CaseLevel: Integration
+        @CaseLevel: System
         """
+        with VirtualMachine(distro='rhel71') as client1:
+            with VirtualMachine(distro='rhel71') as client2:
+                clients = [client1, client2]
+                for client in clients:
+                    client.install_katello_ca()
+                    result = client.register_contenthost(
+                        self.session_org.label,
+                        self.activation_key.name,
+                    )
+                    self.assertEqual(result.return_code, 0)
+                    client.enable_repo(REPOS['rhst7']['id'])
+                    client.install_katello_agent()
+                    client.run(
+                        'yum install -y {0}'.format(FAKE_1_CUSTOM_PACKAGE))
+                with Session(self.browser):
```

Comment:
Thanks for suggestion, i'll apply it. But not in this case, for example - 2 VirtualMachines above can be combined, but Session should be started only when all the actions with those machines are finished successfully.